databricks/koalas/series.py

```diff
@@ -2715,7 +2713,9 @@ def add_prefix(self, prefix) -> "Series":
             ]
             + internal.data_spark_columns
         )
-        return first_series(DataFrame(internal.with_new_sdf(sdf)))
+        return first_series(
+            DataFrame(internal.with_new_sdf(sdf, index_dtypes=([None] * internal.index_level)))
```

Comment:
Why don't we pass `object`s as `index_dtypes` directly?

Reply:
Why `object`?

If we pass `None`, the dtype will be inferred during the initialization of `InternalFrame`.